.github/workflows/release-docs.yml

```
@@ -72,14 +76,16 @@ jobs:
 
               git fetch
 
-              git checkout master
+              git checkout -b feat/add-reference-to-new-changelog-${RELEASE}
 
               # commit and push
               git commit -am "add new changelog file to mkdocs.yml"
           
-              git push origin
+              git push --set-upstream origin feat/add-reference-to-new-changelog-${RELEASE}
           
               echo "Altered mkdocs.yml with new Changelog navigation entry was commited and pushed to origin"
+
+              gh pr create --title "Add reference to new Changelog" --body "Add reference to new Changelog file in mkdocs.yml"
```

Should the PR name be more detailed? Something like that:
```suggestion
              gh pr create --title "Add reference to new Changelog ${RELEASE}" --body "Add reference to new Changelog file in mkdocs.yml"
```

---

Sounds good :+1: